### PR TITLE
symbolics: updating commentaries

### PR DESCRIPTION
**Pull request by btrb**
This PR is for standardizing commentaries.

### CODE REVIEW

File: devito/symbolics/manipulation.py [hunk 1]
```diff
@@ -13,8 +13,8 @@
 
 def freeze(expr):
     """
-    Reconstruct ``expr`` turning all :class:`sympy.Mul` and :class:`sympy.Add`
-    into :class:`FrozenExpr` equivalents.
+    Reconstruct ``expr`` turning all sympy.Mul` and sympy.Add
```
Comment (FabioLuporini) [inline devito/symbolics/manipulation.py]: your forgot a `

Comment (btrb) [reply devito/symbolics/manipulation.py]: oh I have fixed that and forgot to push.

File: devito/finite_differences/coefficients.py [hunk 2]
```diff
@@ -120,6 +120,7 @@ class Substitutions(object):
 
     Now define some partial d/dx FD coefficients of the Function u:
 
+    >>> import numpy as np
```
Comment (FabioLuporini) [inline devito/finite_differences/coefficients.py]: hang on. How can [this test](https://github.com/opesci/devito/blob/master/tests/test_docstrings.py#L27) possibly have worked on CI without this import ? 

How did you add this line? did you notice anything odd?

Comment (btrb) [reply devito/finite_differences/coefficients.py]: Sorry, I didn't understand what you mean.

Comment (btrb) [reply devito/finite_differences/coefficients.py]: I added this line by hand. I went to use this example and noticed that import for `np` was missing.

**Comment by FabioLuporini** [comment]
thanks, merged

**Pull request merged**